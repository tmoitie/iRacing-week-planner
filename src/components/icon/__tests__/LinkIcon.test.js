import React from 'react';
import { render } from '@testing-library/react';
import { describe, test } from '@jest/globals';
import LinkIcon from '../LinkIcon';

describe('components/icon/LinkIcon', () => {
  test('renders correctly', () => {
    const { container } = render(<LinkIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
