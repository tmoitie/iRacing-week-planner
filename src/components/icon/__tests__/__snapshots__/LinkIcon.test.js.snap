// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/icon/LinkIcon renders correctly 1`] = `
<span
  class="glyphicon glyphicon-link"
/>
`;
